pkgs/tools/compression/zstd/default.nix

```diff
@@ -14,56 +15,50 @@ stdenv.mkDerivation rec {
     owner = "facebook";
   };
 
+  outputs = [ "out" "lib" "dev" ];
```

Comment:
This one has potential to break packages.

Reply:
removed.

Reply:
I'm not saying it should not be done. It can be left for later, for sure.